Makefile

```diff
@@ -14,6 +14,5 @@ test-all:
 	karma start --single-run --browsers browserstack-edge
 	karma start --single-run --browsers browserstack-ie11
 	karma start --single-run --browsers browserstack-ie10
-	karma start --single-run --browsers browserstack-ie9
```

Comment:
we don't support ie9 anymore?

Reply:
The time has come! @refractalize thinks so too

Reply:
those poor 3 ie9 users though :cry: